packages/block-library/src/cover/editor.scss

```diff
@@ -38,10 +38,18 @@
 		}
 	}
 
+	.wp-block-cover__placeholder-color-palette {
+		max-width: 300px;
+	}
+
 	// Apply max-width to floated items that have no intrinsic width
 	[data-align="left"] &,
 	[data-align="right"] & {
 		max-width: $content-width / 2;
 		width: 100%;
 	}
 }
+
```

Comment:
@youknowriad I feel like adding styles outside the root selector is bad behavior but I'm not sure how I can do this in a clean way other then injecting a style prop to the component